libs/components/modals/src/lib/modules/modal/modal-host.component.ts

```diff
@@ -139,11 +155,38 @@ export class SkyModalHostComponent implements OnDestroy {
       injector
     );
 
+    // modal element that was just opened
+    const modalElement = modalComponentRef.location.nativeElement;
+
     modalInstance.componentInstance = modalComponentRef.instance;
 
     this.#registerModalInstance(modalInstance);
+    // hidding all elements at the modal-host level from screenreaders
+    const hostElement = this.#elRef.nativeElement;
+    let allModals = hostElement.children;
+
+    if (SkyModalHostService.openModalCount == 1) {
```

Comment:
(I would add a unit test that predicts that exact scenario to confirm.)

Reply:
my thinking was that since the user is in the modal there wouldn't be things added behind it so if we just hide things the first time a modal is opened we would be good. but i guess that's idealic thinking.